models/QtiRunnerInitDataBuilder.php

```diff
@@ -195,7 +203,7 @@ protected function getTestMap(QtiRunnerServiceContext $context, array $itemsStat
                         'skipped' => $isSkipped,
                         'unseen' => $isUnseen,
                         'score' => $itemsStates[$itemId]['score'] ?? null,
-                        'maxScore' => $itemsStates[$itemId]['maxScore'] ?? null // FIXME: unsubmitted item should still have a maxScore
+                        'maxScore' => $itemsStates[$itemId]['maxScore'] ?? (float) $this->getMaxScore($itemData['data']['outcomes'])
```

Comment:
is it possible `$itemData['data']['outcomes']` to not be set here?

Reply:
Created array validation